apps/web/cypress/tests/invites.spec.ts

```diff
@@ -3,21 +3,84 @@ import * as capitalize from 'lodash.capitalize';
 describe('Invites module', function () {
   beforeEach(function () {
     cy.task('clearDatabase');
-    cy.inviteUser('testing-amazing@user.com');
   });
 
   it('should accept invite to organization', function () {
-    doRegister(this.token);
+    cy.inviteUser('testing-amazing@user.com').then(() => {
+      doRegister(this.token);
+
+      cy.getByTestId('header-profile-avatar').click();
+      cy.getByTestId('header-dropdown-organization-name').contains(capitalize(this.organization.name.split(' ')[0]));
+    });
+  });
+
+  it('should allow to login with Github if invited existing user', function () {
+    const isCI = Cypress.env('IS_CI');
+    if (!isCI) return;
```

Comment:
Something we can do in the future is to separate all the CI oriented tests in a specific folder and trigger them only in CI without having to do these checks in the test code.